Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM alpine:3.4
-MAINTAINER Michael de Wit <michael@drillster.com>
+FROM alpine:latest
+MAINTAINER Jesse Quinn <me@jessequinn.info>
 
 RUN apk add --no-cache ca-certificates bash openssh-client rsync
-COPY upload.sh /usr/local/
+COPY drone.sh /usr/local/
```

Comment:
Why did you rename this file? I feel like `upload.sh` was a better match for what the script actually does.

Reply:
renamed as per your request.